virtual_ecosystem/models/litter/inputs.py

```diff
@@ -0,0 +1,349 @@
+"""The ``models.litter.inputs`` module handles the partitioning of plant matter into the
+various pools of the litter model. This plant matter comes from both natural tissue
+death as well as from mechanical inefficiencies in herbivory.
+"""  # noqa: D205
+
+from __future__ import annotations
+
+from dataclasses import dataclass
+
+import numpy as np
+from numpy.typing import NDArray
+
+from virtual_ecosystem.core.data import Data
+from virtual_ecosystem.core.logger import LOGGER
+from virtual_ecosystem.models.litter.constants import LitterConsts
+
+
+@dataclass
```

Comment:
If the `LitterInputs` objects are not to be modified after creation, make it frozen with `@dataclass(frozen=True)` to avoid mistakes.